ravelin.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://developer.ravelin.com'
   spec.license       = 'MIT'
 
-  spec.add_dependency('faraday', '~> 0.9')
+  spec.add_dependency('faraday', '~> 0.15.4')
   spec.add_dependency('faraday_middleware', '~> 0.10')
```

Comment:
Is there any form of dependency between `faraday` and its `middleware`?

Reply:
Don't think so - they're working together here at least